src/hexdoc/_templates/pages/patchouli/entity.html.jinja

```diff
@@ -0,0 +1,21 @@
+{% extends "pages/patchouli/page.html.jinja" %}
+
+{% import "macros/formatting.html.jinja" as fmt with context %}
+{% import "macros/textures.html.jinja" as texture_macros with context %}
+
+{% block body %}
+<h4>
+  {{- page.get_name() -}}
+  {%- if page_anchor_id is defined %}
+    {{- fmt.permalink(page_anchor_id) -}}
+  {% endif -%}
+</h4>
+<p class="image-wrapper">
```

Comment:
The image should probably be wrapped in a block so users can override the entity display if they want.